packages/changed-elements-react/src/clients/iTwinApi.ts

```diff
@@ -16,15 +15,23 @@ export interface CallITwinApiParams {
 export async function callITwinApi(args: CallITwinApiParams): Promise<Record<string, unknown>> {
   const response = await fetch(
     args.url,
-    {
+    args.getAccessToken ? {
```

Comment:
Token is not needed when getting changed elements from Href. It won't send the request correctly. I could make another method for the href case.

  public async getComparisonJobResult(args: GetComparisonJobResultParams): Promise<ChangedElements> {
    return callITwinApi({
      url: args.comparisonJob.comparison.href,
      method: "GET",
      signal: args.signal,
      headers: {
        Accept: ComparisonJobClient._acceptHeader,
        ...args.headers,
      },
    }) as unknown as Promise<ChangedElements>;
  }

Reply:
iTwin Platform API calls always require an auth token. It seems that `getComparisonJobResult` is not actually making a call to iTwin Platform thus you should just use plain `fetch` there.

Reply:
The fetch for the HREF shouldn't require any headers, because the authorization is in the URL itself, so you could remove this logic if all you are doing here is downloading the blob data.